### PR TITLE
dashboard: add account receiver creation to integration tests

### PR DESCRIPTION
**Pull request by chrisgarvin**

### CODE REVIEW

File: dashboard/test/conf/wdio.local.js [hunk 1]
```diff
@@ -43,7 +43,7 @@ exports.config = {
   // and 30 processes will get spawned. The property handles how many capabilities
   // from the same test should run tests.
   //
-  maxInstances: extended ? 1 : 10,
+  maxInstances: extended ? 1 : 1,
```
Comment (dominic) [inline dashboard/test/conf/wdio.local.js]: This can just be `maxInstances: 1`

File: dashboard/test/helpers.js [hunk 2]
```diff
@@ -84,10 +85,25 @@ const issueTransaction = (client) => expect(
   .then(tpl => client.transactions.submit(tpl))
 ).to.be.fulfilled
 
+const createAccount = (account = 'account') => {
```
Comment (jeffomatic) [inline dashboard/test/helpers.js]: maybe rename `account` to `name`?

Comment (chrisgarvin) [inline dashboard/test/helpers.js]: yeah, good call

File: dashboard/test/specs/accounts.js [hunk 3]
```diff
@@ -31,13 +31,41 @@ describe('accounts', () => {
...
+  describe('creating receivers', () => {
+    let account
+
+    before(() => expect(testHelpers.ensureConfigured()).to.be.fulfilled
+      .then(() => {
+        return testHelpers.createAccount('receiver-test')
+      })
+      .then((acc) => {
+        account = acc
+      })
```
Comment (dominic) [inline dashboard/test/specs/accounts.js]: This doesn't need to be a second `then`.

File: dashboard/test/specs/accounts.js [hunk 4]
```diff
@@ -31,13 +31,41 @@ describe('accounts', () => {
...
       browser.getText('.AccountShow').should.contain(alias)
     })
   })
+
+  describe('creating receivers', () => {
+    let account
+
+    before(() => expect(testHelpers.ensureConfigured()).to.be.fulfilled
+      .then(() => {
+        return testHelpers.createAccount('receiver-test')
```
Comment (dominic) [inline dashboard/test/specs/accounts.js]: You use implicit return at the end of this `then` chain; let's stick with that for one liners.

File: dashboard/test/specs/accounts.js [hunk 5]
```diff
@@ -31,13 +31,32 @@ describe('accounts', () => {
...
       browser.click('.FormContainer button')
       browser.waitForVisible('.AccountShow')
       browser.getText('.AccountShow').should.contain('Created account. Create another?')
       browser.getText('.AccountShow').should.contain(alias)
     })
   })
+
+  describe('creating receivers', () => {
+    before(() => expect(testHelpers.ensureConfigured()).to.be.fulfilled
+      .then(() => testHelpers.createAccount('receiver-test'))
```
Comment (dominic) [inline dashboard/test/specs/accounts.js]: This will fail on a second run - can we add a UUID?

**Pull request merged**